rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2022-06-16
```

Comment:
This can be a `rust-toolchain.toml` file https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file

Can specify the `wasm32-unknown-unknown` target so that the `build.sh` above doesn't need it.

Reply:
@chinedufn specified default target in .cargo/config file